Gruntfile.js

```diff
@@ -833,6 +833,14 @@ module.exports = function(grunt) {
       'build/ddocs/medic-admin/_attachments/js/templates.js':
         'build/ddocs/medic-admin/_attachments/js/templates.js',
     },
+    jsdoc : {
+      dist : {
+          src: ['webapp/src/js/*.js', 'admin/src/js/*.js'],
```

Comment:
When I run this the index.html is empty. I think this should be `['webapp/src/js/**/*.js', 'admin/src/js/**/*.js']` so it recurses into subdirectories.